Makefile

```diff
@@ -3,7 +3,7 @@
 # example: "make setup"
 
 PYTHON=python
-LINT_DIRS=ibm_platform_services test examples
+LINT_DIRS=ibm_platform_services test/unit test/integration examples
```

Comment:
pylint expects modules to be specified, so we were never really lint-checking unit or integration tests before :joy:

Reply:
Wow, nice catch!